addon/utils/notes-placement.js

```diff
@@ -0,0 +1,32 @@
+export function getNotesIdealPlacement(notes, activeNoteId, gutter) {
+  const activeNote = notes.find(({ id }) => activeNoteId === id) || notes[0];
+  const activeNoteIndex = notes.indexOf(activeNote);
```

Comment:
```suggestion
  const activeNoteIndex = notes.findIndex(({ id }) => activeNoteId === id) || 0;
```

Reply:
`findIndex` returns -1 not a falsy value

Reply:
True true true